src/wp-includes/rss.php

```diff
@@ -338,7 +338,7 @@ function append($el, $text) {
 	function normalize () {
 		// if atom populate rss fields
 		if ( $this->is_atom() ) {
-			$this->channel['descripton'] = $this->channel['tagline'];
+			$this->channel['description'] = $this->channel['tagline'];
```

Comment:
Do this changes make any issue for old versions?

Reply:
Based on the discussion at [#57102](https://core.trac.wordpress.org/ticket/57102) please check my reply for the same:

https://core.trac.wordpress.org/ticket/57102#comment:3